unittests/test_schedulers.py

```diff
@@ -655,7 +668,7 @@ def test_cancel_with_grace(minimal_job, scheduler, local_only):
 
     # Verify that the spawned sleep is killed, too, but back off a bit in
     # order to allow the sleep process to wake up and get the signal
-    time.sleep(0.1)
+    time.sleep(5)
```

Comment:
I still can't reproduce this locally - the tests always pass. I'm assuming there is some delay in the CI container and testing adding a longer sleep here.

Reply:
Nope that didn't work! @vkarak do you have any ideas? I can't reproduce this error locally.

Reply:
Check my comment in your Dockerfile. I found another bug, which I don't know if it's related to this failure, but let's first fix that.